From e0fbd55475a5b3f1e98d3a28b329e0a22089b962 Mon Sep 17 00:00:00 2001
From: Xinyu Chen <xinyu.chen@freescale.com>
Date: Thu, 14 Aug 2008 16:00:38 +0800
Subject: [PATCH] ENGR00087892 MX35 Workaround solution to reduce video playback bus loading

Turn down the IPU DI pixel clock from 25MHz to 20MHz.
It makes the LCD refresh rate to 44Hz

Signed-off-by: Xinyu Chen <xinyu.chen@freescale.com>
---
 drivers/video/mxc/mxcfb_claa_wvga.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/video/mxc/mxcfb_claa_wvga.c b/drivers/video/mxc/mxcfb_claa_wvga.c
index a9e18c4..2c1435e 100644
--- a/drivers/video/mxc/mxcfb_claa_wvga.c
+++ b/drivers/video/mxc/mxcfb_claa_wvga.c
@@ -48,7 +48,7 @@ static int lcd_on;
 static struct fb_videomode video_modes[] = {
 	{
 	 /* 800x480 @ 55 Hz , pixel clk @ 25MHz */
-	 "CLAA-WVGA", 55, 800, 480, 40000, 80, 0, 10, 0, 20, 10,
+	 "CLAA-WVGA", 55, 800, 480, 50000, 80, 0, 10, 0, 20, 10,
 	 FB_SYNC_OE_ACT_HIGH,
 	 FB_VMODE_NONINTERLACED,
 	 0,},
-- 
1.5.4.4

